localisation/es-es/docs/01-monkey-app.md

```diff
@@ -0,0 +1,164 @@
+    ```
+
+1. Ingresa el siguiente prompt a GitHub Copilot para generar un issue en el repositorio.
+
+    ```text
+    Create a new GitHub issue in my repository titled 'Implement Monkey Console Application' with the following requirements:
+    
+    - Create a C# console app that can list all available monkeys, get details for a specific monkey by name, and pick a random monkey.
+    - The app should use a Monkey model class and include ASCII art for visual appeal.
+    - Add appropriate labels like 'enhancement' and 'good first issue'.
+    - Add some details about how we may go about implementing this and a checklist for what we will need to do.
+    ```
+
+1. Asigna `@Copilot` al issue y observa lo que está sucediendo.
+
+## Iniciar Servidor MCP &ndash; Monkey MCP
+
+1. Abre la Paleta de Comandos presionando `F1` o `Ctrl`+`Shift`+`P` en Windows o `Cmd`+`Shift`+`P` en Mac OS, y busca `MCP: List Servers`.
+1. Asegúrate de que el servidor MCP `github` esté funcionando.
+1. Elige `monkeymcp` luego haz clic en `Start Server`.
+
+## Desarrollar Aplicación Monkey con GitHub Copilot y Servidores MCP
+
+1. Ingresa el siguiente prompt para obtener la lista de monos.
+
+    ```text
+    Get me a list of monkeys that are available and display them in a table with their details.
+    ```
+
+1. Ingresa el siguiente prompt para obtener una idea del modelo de datos para un mono.
+
+    ```text
+    What would a data model look like for this structure?
+    ```
+
+1. Ingresa el siguiente prompt para crear un archivo para la clase del modelo de datos.
+
+    ```text
+    Let's create a new file for this class.
+    ```
+
+1. Ingresa el siguiente prompt para crear una clase `MonkeyHelper`.
+
+    ```text
+    Let's create a new class called MonkeyHelper that is static. It should manage a collection of monkey data. Include methods to get all monkeys, get a random monkey, find a monkey by name, and track access count to when a random monkey is picked. The data for the monkeys should come from the Monkey MCP server that we just got.
+    ```
+
+1. Ingresa el siguiente prompt para actualizar la aplicación de consola.
+
+    ```text
+    Let's update the app now to have a nice menu with the following options that will call into that `MonkeyHelper`.
+    
+    1. List all monkeys
+    2. Get details for a specific monkey by name
+    3. Get a random monkey
+    4. Exit app
+
+    Also display some funny ASCII art randomly.
+    ```
+
+1. Ingresa el siguiente prompt a GitHub Copilot para subir la aplicación de consola actualizada.
+
+    ```text
+    Push the current changes to the `mymonkeyapp` branch of the repository.
+    With this branch, create a PR against the `main` branch.
+    Connect this PR to the issue created before.
+    Then, merge this PR and close the issue.
+    ```
+
+---
+
+OK. Has completado el paso "Desarrollo de Aplicación Monkey con MCP". Continuemos con [PASO 02: Servidor MCP](./02-mcp-server.md).
```

Comment:
Make sure that the disclaimer should be included at the end of the localised document.